site/content/in-dev/unreleased/quickstart.md

```diff
@@ -97,42 +97,52 @@ To start using Polaris in Docker, launch Polaris while Docker is running:
+docker run -p 8181:8181 -p 8182:8182 apache/polaris:latest
 ```
 
-Once the `polaris-polaris` container is up, you can continue to [Defining a Catalog](#defining-a-catalog).
+You should see output for some time as Polaris builds and starts up. Eventually, you won’t see any more logs and should see messages that resemble the following:
+
+```
+INFO  [io.quarkus] [,] [,,,] (Quarkus Main Thread) Apache Polaris Server <version> on JVM  (powered by Quarkus <version>) started in 2.656s. Listening on: http://localhost:8181. Management interface listening on http://0.0.0.0:8182.
+INFO  [io.quarkus] [,] [,,,] (Quarkus Main Thread) Profile prod activated.
+INFO  [io.quarkus] [,] [,,,] (Quarkus Main Thread) Installed features: [...]
+```
+
+Once the container is up, you can continue to [Defining a Catalog](#defining-a-catalog).
 
 ### Building Polaris
 
-Run Polaris locally with:
+Run Polaris locally in [Quarkus Dev mode](https://quarkus.io/guides/dev-mode-differences) with:
 
 ```shell
 cd ~/polaris
-./gradlew runApp
+./gradlew --console=plain :polaris-quarkus-service:quarkusDev
 ```
 
 You should see output for some time as Polaris builds and starts up. Eventually, you won’t see any more logs and should see messages that resemble the following:
 
 ```
-INFO  [...] [main] [] o.e.j.s.handler.ContextHandler: Started i.d.j.MutableServletContextHandler@...
-INFO  [...] [main] [] o.e.j.server.AbstractConnector: Started application@...
-INFO  [...] [main] [] o.e.j.server.AbstractConnector: Started admin@...
-INFO  [...] [main] [] o.eclipse.jetty.server.Server: Started Server@...
+realm: POLARIS root principal credentials: <client-id>:<client-secret>
+INFO  [io.quarkus] [,] [,,,] (Quarkus Main Thread) polaris-quarkus-service 1.0.0-incubating-SNAPSHOT on JVM (powered by Quarkus 3.17.6) started in 2.656s. Listening on: http://localhost:8181. Management interface listening on http://0.0.0.0:8182.
+INFO  [io.quarkus] [,] [,,,] (Quarkus Main Thread) Profile dev activated. Live Coding activated.
+INFO  [io.quarkus] [,] [,,,] (Quarkus Main Thread) Installed features: [...]
 ```
 
-At this point, Polaris is running.
+At this point, Polaris is running. Take note of the root principal credentials provided in the logs, as we'll use them in the next section.
 
 ## Bootstrapping Polaris
 
 For this tutorial, we'll launch an instance of Polaris that stores entities only in-memory. This means that any entities that you define will be destroyed when Polaris is shut down. It also means that Polaris will automatically bootstrap itself with root credentials. For more information on how to configure Polaris for production usage, see the [docs]({{% ref "configuring-polaris-for-production" %}}).
 
-When Polaris is launched using in-memory mode the root principal credentials can be found in stdout on initial startup. For example:
+When Polaris is launched using an in-memory metastore, such as when Quarkus Dev mode is used, the
```

Comment:
Do end users care about Quarkus to the point that we need to mention "Quarkus Dev mode"?

If nothing else, couldn't we just say "dev mode"?

Reply:
Sure, I can change that.